src/main/fulcro/client/primitives.cljc

```diff
@@ -3228,6 +3237,14 @@
     "The state has to be an atom. Use 'integrate-ident' instead.")
   (apply swap! state integrate-ident ident named-parameters))
 
+(defn remove-ident!
```

Comment:
Move to mutations, and make the signature:

`(remove-ident! this field ident)`

so it can be used in a component that owns a list instead of writing a transact.